library/src/main/java/xyz/klinker/android/article/ImageViewActivity.java

```diff
@@ -16,31 +16,41 @@
 
 package xyz.klinker.android.article;
 
+import android.graphics.Color;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
 import android.widget.ImageView;
 
 import com.bumptech.glide.Glide;
 import com.bumptech.glide.load.engine.DiskCacheStrategy;
 
+import xyz.klinker.android.drag_dismiss.activity.DragDismissActivity;
+
 /**
  * Activity for viewing full size images from an article. Images are zoomable. You can pass in the
  * image url from an intent with .setData(url).
  */
-public final class ImageViewActivity extends AppCompatActivity {
+public final class ImageViewActivity extends DragDismissActivity {
```

Comment:
Yes, I did the same in talon and feel the same way. Should be fairly trivial to add this, will do